packages/sources/src/api/createSource.js

```diff
@@ -78,6 +79,11 @@ export const doCreateSource = async (formData, sourceTypes) => {
 
         const [ endpointDataOut, applicationDataOut ] = await Promise.all(promises);
 
+        if (formData.source_type === 'azure') {
+            console.log('lets get this party started');
+            await patchSource({ id: sourceDataOut.id, ...formData.cm_source });
```

Comment:
you are still using cm_source here, it should be something like `{billing_source: formData.billing_source, authentication: formData.credentials }`